docs/mkdocs.yml

```diff
@@ -13,9 +13,9 @@ nav:
       - Install Datapane: "tutorials/basics/install-datapane.md"
       - Create a report: "tutorials/basics/create-a-report.ipynb"
     - 🧱 Blocks types:
-      - tutorials/blocks/index.ipynb
+      - Block types: tutorials/blocks/index.ipynb
```

Comment:
Interesting change do the section indexes in `mkdocs.yml`! We had followed the [docs](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#section-index-pages) for this, is this a better way of doing it?

Reply:
I found the `.ipynb` files weren't setting their page titles (eg [blocks](https://docs.datapane.com/catalogues/blocks/)) - a bug in  ``mknotebooks`` perhaps? The easiest way to fix it seemed to be by setting the index page titles manually.